backend/app/tests/utils/test_jsonl_parser.py

```
@@ -0,0 +1,432 @@
+        
+        options = ParseOptions(preprocessor=preprocessor_config)
+        result = await parser.parse(content, options)
+        
+        assert result.success
+        assert len(result.blocks) == 2
+
+    async def test_error_limit(self, parser):
+        """测试错误限制"""
+        # 创建包含多个错误的内容
+        lines = []
+        for i in range(20):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)